host/host_aix.go

```diff
@@ -68,15 +68,16 @@ func parseUptime(uptime string) uint64 {
 	var err error
 
 	switch ut[3] {
-	case "day,", "days,":
+	case "day,", "days,", "day", "days":
 		days, err = strconv.ParseUint(ut[2], 10, 64)
 		if err != nil {
 			return 0
 		}
 
 		// day provided along with a single hour or hours
 		// ie: up 2 days, 20 hrs,
-		if ut[5] == "hr," || ut[5] == "hrs," {
+		if ut[5] == "hr," || ut[5] == "hrs," ||
```

Comment:
This needs a subhandling for mins to be in ut[7], for a case where you have 20 days, 17 hrs, 5 mins

it all of these ifs should likely be exists/nil protected as well, since these array elements will only exist if the returned value previously has enough elements to fill it out this far.

Also both of these if ut[5] should probably be another switch/case statement for consistency.